pycfmodel/model/cf_model.py

```diff
@@ -22,31 +22,26 @@ def __init__(self, cf_script):
         self.aws_template_format_version = cf_script.get("AWSTemplateFormatVersion")
         self.description = cf_script.get("Description")
         self.metadata = cf_script.get("Metadata")
+
```

Comment:
Is this space separating something in particular or can it be removed?

Reply:
It's to separate variables that store info about the CF, from the ones that store the information created by the cloudformation.